docs/man-openmpi/man3/MPI_File_seek.3.rst

```diff
@@ -66,11 +66,9 @@ DESCRIPTION
 :ref:`MPI_File_seek` updates the individual file pointer according to *whence,*
 which could have the following possible values:
 
- o    MPI_SEEK_SET - The pointer is set to *offset.*
-
- o    MPI_SEEK_CUR - The pointer is set to the current pointer position plus *offset.*
-
- o    MPI_SEEK_END - The pointer is set to the end of the file plus *offset.*
+* MPI_SEEK_SET - The pointer is set to *offset.*
+* MPI_SEEK_CUR - The pointer is set to the current pointer position plus *offset.*
+* MPI_SEEK_END - The pointer is set to the end of the file plus *offset.*
```

Comment:
The `MPI_SEEK_*` values are constants: they should probably be in fixed-width, right? (in all the man pages they appear)

Reply:
I have now amended, please have a look if you think they are correct (once CI is done)